pkg/apis/operator/v1alpha1/tektonresult_types.go

```diff
@@ -53,26 +53,32 @@ func (tp *TektonResult) GetStatus() TektonComponentStatus {
 type TektonResultSpec struct {
 	CommonSpec           `json:",inline"`
 	ResultsAPIProperties `json:",inline"`
+	LokiStackProperties  `json:",inline"`
+}
+
+type LokiStackProperties struct {
+	LokiStackName      string `json:"loki_stack_name,omitempty"`
+	LokiStackNamespace string `json:"loki_stack_namespace,omitempty"`
 }
 
 // ResultsAPIProperties defines the fields which are configurable for
 // Results API server config
 type ResultsAPIProperties struct {
 	DBHost                string `json:"db_host,omitempty"`
-	DBPort                int64  `json:"db_port,omitempty"`
+	DBPort                *int64 `json:"db_port,omitempty"`
```

Comment:
Any reason why are we changing it to pointers 
Won't this break upgrade ?
cc @jkandasa

Reply:
@khrm since we have modified too may CR properties type, please perform a through upgrade tests and report back the status of the tests. Thanks!

Reply:
Reverse is the breaking. Changing from pointer to concrete values.

Reply:
The reason for changing these values to pointers is so that we don't force users to set these values. Otherwise, zero values of these are taken as values given by the user.